index.html

```diff
@@ -300,85 +300,10 @@ <h3>
         </h3>
         <p>
           The {{Document/onvisibilitychange}} attribute is an <a>event handler
-          IDL attribute</a> for the `"visibilitychange"` event type (see
-          [[[#reacting-to-visibilitychange]]]).
+          IDL attribute</a> for the `"visibilitychange"` event type.
```

Comment:
We probably want to have a note saying that the event is fired by the html spec (if the spec doesn’t already state that).

Reply:
Fixed